pkg/virt-handler/vm.go

```diff
@@ -2607,6 +2607,10 @@ func (d *VirtualMachineController) vmUpdateHelperDefault(origVMI *v1.VirtualMach
 			}
 		}
 
+		if err := d.ensureDVsOwnership(vmi); err != nil {
+			return err
+		}
```

Comment:
what is specific about DataVolume's here? i thought this was specific to PVCs, which may be referened by a DataVolume or PVC volume source on the vmi.

Reply:
Datavolumes are handled by CDI and therefore theoretically we don't need to do anything. But CDI is using fsgroup which is not supported by all storage. Ideally we move this responsibility to CDI(imho).

Reply:
i think my question was more trying to point out that this function isn't DV specific. Its primary function is to ensure ownership of PVCs, right?

Reply:
Correct. I renamed it but I am opened to a better suggestion.